src/newcharacter.cpp

```diff
@@ -2428,6 +2429,10 @@ void save_template( player *u )
     if( name.length() == 0 ) {
         return;
     }
+    if( name.find( '/' ) != std::string::npos || name.find( '\\' ) != std::string::npos ) {
+        name.clear();
+        name.insert( 0, std::to_string( ( unsigned long long ) time( nullptr ) ) );
```

Comment:
This is a very strange way to save `name = std::to_string( ... );`.

You could show a `popup` telling the use about the invalid characters and than just leave the function.